apply_extra.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -e
+shopt -s failglob
+FLATPAK_ID="${FLATPAK_ID:-cn.wps.wps_365}"
+CARCH="$(uname -m)"
+
+mkdir -p export/share/
+
+bsdtar --to-stdout -xf wps-office.deb data.tar.xz | bsdtar -xf -
+
+# Move shared files to export directory
+mv usr/share/{icons,applications,mime} export/share/
+
+# Rename icons, desktop files, and mime files to use Flatpak ID
+YEAR_SUFFIX=2023
+rename --no-overwrite "wps-office-" "${FLATPAK_ID}." export/share/{icons/hicolor/*/*,applications}/wps-office-*.*
+rename --no-overwrite "wps-office${YEAR_SUFFIX}-" "${FLATPAK_ID}." export/share/icons/hicolor/*/*/wps-office${YEAR_SUFFIX}-*.*
+rename --no-overwrite "custom-wps-office" "${FLATPAK_ID}" export/share/mime/packages/custom-wps-office.xml
+rename --no-overwrite "xiezuo" "${FLATPAK_ID}.xiezuo" export/share/{icons/hicolor/*/*,applications}/xiezuo.*
+
+# Edit .desktop files to adjust Exec and Icon entries
+for a in wps wpp et pdf prometheus xiezuo; do
+    desktop_file="export/share/applications/${FLATPAK_ID}.$a.desktop"
+    appbin="$a %F"
+    appicon="${FLATPAK_ID}.${a}main"
+    renamedfrom="wps-office-$a.desktop"
+    case "$a" in
+        pdf)
+            appbin="wpspdf %F"
+        ;;
+        prometheus)
+            appbin="wps %F"
+            appicon="${FLATPAK_ID}.k${a}"
+            # Use this as the main .desktop file for the Flatpak
+            new_desktop_file="$(dirname $desktop_file)/${FLATPAK_ID}.desktop"
+            mv $desktop_file $new_desktop_file
+            desktop_file=$new_desktop_file
+        ;;
+        xiezuo)
+            appbin="/app/extra/opt/xiezuo/xiezuo --no-sandbox --disable-gpu-sandbox --disable-setuid-sandbox --package-format=deb %U"
+            appicon="${FLATPAK_ID}.xiezuo"
+            renamedfrom="xiezuo.desktop"
+        ;;
+    esac
+    sed -i 's|Categories=.*|&Office;|' "$desktop_file"
+    desktop-file-edit \
+        --set-key="Exec" --set-value="$appbin" \
+        --set-key="Icon" --set-value="$appicon" \
+        --set-key="X-Flatpak-RenamedFrom" --set-value="$renamedfrom" \
+        "$desktop_file"
```

Comment:
why? just modify the desktop file you added to the PR directly? 

also you do not need to add `X-Flatpak-RenamedFrom`